terminal-here.el

```diff
@@ -30,6 +30,12 @@
   :group 'external
   :prefix "terminal-here-")
 
+(defun terminal-here-find-executable (variants)
+  (file-name-nondirectory
+   (cl-some (lambda (executable)
+              (executable-find executable))
```

Comment:
Is this different to `(cl-some #'executable-find variants)`? It looks like that `lambda` doesn't do anything.